[2025-03-24T17:25:15.686+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T17:25:15.693+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e2e_data_pipeline.dbt_execute manual__2025-03-24T16:17:51.573790+00:00 [queued]>
[2025-03-24T17:25:15.696+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e2e_data_pipeline.dbt_execute manual__2025-03-24T16:17:51.573790+00:00 [queued]>
[2025-03-24T17:25:15.696+0000] {taskinstance.py:2867} INFO - Starting attempt 13 of 13
[2025-03-24T17:25:15.699+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_execute> on 2025-03-24 16:17:51.573790+00:00
[2025-03-24T17:25:15.702+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'e2e_data_pipeline', 'dbt_execute', 'manual__2025-03-24T16:17:51.573790+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpryf7wkd1']
[2025-03-24T17:25:15.703+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask dbt_execute
[2025-03-24T17:25:15.703+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1079) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-24T17:25:15.703+0000] {standard_task_runner.py:72} INFO - Started process 1080 to run task
[2025-03-24T17:25:15.719+0000] {task_command.py:467} INFO - Running <TaskInstance: e2e_data_pipeline.dbt_execute manual__2025-03-24T16:17:51.573790+00:00 [running]> on host 5534f92a2ac4
[2025-03-24T17:25:15.744+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='e2e_data_pipeline' AIRFLOW_CTX_TASK_ID='dbt_execute' AIRFLOW_CTX_EXECUTION_DATE='2025-03-24T16:17:51.573790+00:00' AIRFLOW_CTX_TRY_NUMBER='13' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-24T16:17:51.573790+00:00'
[2025-03-24T17:25:15.744+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T17:25:15.748+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T17:25:15.749+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'docker exec data_pipeline-dbt-1 dbt run']
[2025-03-24T17:25:15.752+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T17:25:16.649+0000] {subprocess.py:106} INFO - [0m17:25:16  Running with dbt=1.9.0
[2025-03-24T17:25:16.805+0000] {subprocess.py:106} INFO - [0m17:25:16  Registered adapter: postgres=1.9.0
[2025-03-24T17:25:17.076+0000] {subprocess.py:106} INFO - [0m17:25:17  Found 3 models, 3 data tests, 1 source, 430 macros
[2025-03-24T17:25:17.078+0000] {subprocess.py:106} INFO - [0m17:25:17
[2025-03-24T17:25:17.079+0000] {subprocess.py:106} INFO - [0m17:25:17  Concurrency: 1 threads (target='dev')
[2025-03-24T17:25:17.080+0000] {subprocess.py:106} INFO - [0m17:25:17
[2025-03-24T17:25:17.211+0000] {subprocess.py:106} INFO - [0m17:25:17  1 of 3 START sql view model public.stg_pl_2425 ................................. [RUN]
[2025-03-24T17:25:17.279+0000] {subprocess.py:106} INFO - [0m17:25:17  1 of 3 OK created sql view model public.stg_pl_2425 ............................ [[32mCREATE VIEW[0m in 0.07s]
[2025-03-24T17:25:17.281+0000] {subprocess.py:106} INFO - [0m17:25:17  2 of 3 START sql table model public.pl_2425_matches ............................ [RUN]
[2025-03-24T17:25:17.322+0000] {subprocess.py:106} INFO - [0m17:25:17  2 of 3 OK created sql table model public.pl_2425_matches ....................... [[32mSELECT 380[0m in 0.04s]
[2025-03-24T17:25:17.323+0000] {subprocess.py:106} INFO - [0m17:25:17  3 of 3 START sql table model public.pl_2425_standings .......................... [RUN]
[2025-03-24T17:25:17.349+0000] {subprocess.py:106} INFO - [0m17:25:17  3 of 3 OK created sql table model public.pl_2425_standings ..................... [[32mSELECT 20[0m in 0.02s]
[2025-03-24T17:25:17.357+0000] {subprocess.py:106} INFO - [0m17:25:17
[2025-03-24T17:25:17.358+0000] {subprocess.py:106} INFO - [0m17:25:17  Finished running 2 table models, 1 view model in 0 hours 0 minutes and 0.28 seconds (0.28s).
[2025-03-24T17:25:17.383+0000] {subprocess.py:106} INFO - [0m17:25:17
[2025-03-24T17:25:17.383+0000] {subprocess.py:106} INFO - [0m17:25:17  [32mCompleted successfully[0m
[2025-03-24T17:25:17.383+0000] {subprocess.py:106} INFO - [0m17:25:17
[2025-03-24T17:25:17.384+0000] {subprocess.py:106} INFO - [0m17:25:17  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-03-24T17:25:18.726+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T17:25:18.737+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T17:25:18.737+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=e2e_data_pipeline, task_id=dbt_execute, run_id=manual__2025-03-24T16:17:51.573790+00:00, execution_date=20250324T161751, start_date=20250324T172515, end_date=20250324T172518
[2025-03-24T17:25:18.764+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T17:25:18.771+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-24T17:25:18.771+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
